Created README.md (#23)
Terasology/Lost: README.md

# Lost

Are you truly lost? Will you ever return home?
A researcher on a space expedition is stranded on an alien world with no way to get home.  

>  ## How To Play

Reviewer: Consider adding a note on that the structures (the small house and the puzzle structures to find when exploring) are generated during the game. So sometimes visiting a location twice can be useful. Also, - in case it's not in the book of secrets texts (I'm not sure about that) - mentioning that different times of day might help to find different puzzles might make sense.

You can use collapsible text (like in https://gist.github.com/joyrexus/16041f2426450e73f5df9391f7f7ae5f) to hide potential spoilers.

Author: ok will take note.

> Download the Lost module and it's dependacies. When this is installed, create a new world with the Lost template. If everything worked, you should spawn in a world with a small house nearby.

Reviewer: ```suggestion
Download the Lost module and it's dependencies. When this is installed, create a new world with the Lost template. If everything worked, you should spawn in a world with a small house nearby.
```

Author: ahh I knew I would misspell something!

## Current Features

### Special Structure Generation

> Providing the story of the Lost World.

Reviewer: Not sure what you mean here?

Author: I was just guessing, I haven't tested Lost enough to know about the set pieces that spawn.

Author: Replacing section with better details

# Lost

Are you truly lost? Will you ever return home?
A researcher on a space expedition is stranded on an alien world with no way to get home.  

 ## How To Play

> Download the Lost module and it's dependencies. When this is installed, create a new world with the Lost template. If everything worked, you should spawn in a world with a small house nearby.

Reviewer 2: "It's" vs "Its"

```suggestion
Download the Lost module and its dependencies. When this is installed, create a new world with the Lost template. If everything worked, you should spawn in a world with a small house nearby.
```

## Current Features

### Special Structure Generation

> As you explore you might find buildings that contain treasures, secerts and puzzels.

Reviewer 2: Another typo. It really helps to use an editor that automatically checks for spelling and even grammar / issues like the its vs it's bit - even if you're editing in a browser you can usually get addons for that :-)

```suggestion
As you explore you might find buildings that contain treasures, secrets and puzzles.
```
